packages/web-service/src/pages/returns/returns-check.njk

```diff
@@ -72,7 +72,7 @@
         <dt class="govuk-summary-list__key">When did you start the work?</dt>
           <dd class="govuk-summary-list__value">{{ data.startDate }}</dd>
         <dd class="govuk-summary-list__actions">
-          <a class="govuk-link" href="/work-start">
+          <a class="govuk-link" href="/date-work-started">
```

Comment:
should be from the config ? not hard coded. 

see - https://github.com/DEFRA/wildlife-licencing/pull/668/files#diff-8535aa6edda0333eb8c59517d56c9b517cba5ea53513f6e04b9812c40076daddR72